api_compatibility_policy.md

```diff
@@ -95,20 +94,23 @@ See the current list of [alpha features](https://github.com/tektoncd/pipeline/bl
 
 - Alpha feature in beta or GA CRDs are disabled by default and must be enabled by [setting `enable-api-fields` to `alpha`](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features)
 
-- These features may be dropped or backwards incompatible changes made at any time though will be given at least one release worth of warning
+- These features may be dropped or backwards incompatible changes made at any time, though one release worth of warning will be provided.
 
 - Alpha features are reviewed for promotion to beta at a regular basis. However, there is no guarantee that they will be promoted to beta.
 
 ### Beta features
 
 - Beta features in GA CRDs are disabled by default and must be enabled by [setting `enable-api-fields` to `beta`](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#beta-features). In beta API versions, beta features are enabled by default.
 
-- Beta features may be deprecated or changed in a backwards incompatible way by following the same process as [Beta CRDs](#beta-crds) 
+- These features will not be dropped, though the details may change.
+
+- Beta features may be changed in a backwards incompatible way by following the same process as [Beta CRDs](#beta-crds) 
   i.e. by providing a 9 month support period.
 
 - Beta features are reviewed for promotion to GA/Stable on a regular basis. However, there is no guarantee that they will be promoted to GA/stable.
 
 - For beta API versions, beta is the highest level of stability possible for any feature.
+TODO(#6592): Decouple feature stability from API stability.
```

Comment:
Just a question. Since this is user(developer) facing doc, what will be changed in this doc after #6592 is done? I'm just thinking about whether we should inform some possible changes with the decoupling. Or maybe not.

Reply:
I think you're right that this sentence is a bit confusing here; I've removed it